src/prefect/flows.py

```diff
@@ -946,10 +947,14 @@ def my_flow(name):
             else:
                 raise
 
-        if loop is not None:
-            loop.run_until_complete(runner.start(webserver=webserver))
-        else:
-            asyncio.run(runner.start(webserver=webserver))
+        try:
+            if loop is not None:
+                loop.run_until_complete(runner.start(webserver=webserver))
+            else:
+                asyncio.run(runner.start(webserver=webserver))
+        except (KeyboardInterrupt, TerminationSignal):
```

Comment:
Could we maybe log _something_ here so that it's clear why deployments are being paused/etc. Something like "Received {type(exc)}, shutting down..." with no traceback.

Reply:
makes sense, it bothers me the tiniest bit that this almost seems out of order (i.e itd be nice if we could say "Received {exc}" _and then_ show the side effects (pausing schedule etc)) but as is I think this is fine

```
^C11:28:14.973 | INFO    | prefect.runner - Pausing all deployments...
11:28:15.165 | INFO    | prefect.runner - All deployments have been paused!
11:28:15.166 | INFO    | prefect.flows - Received KeyboardInterrupt, shutting down...
```

updated in [e3b10d1](https://github.com/PrefectHQ/prefect/pull/14848/commits/e3b10d1b70e4cb4e67e0950f0b4e3e47dec6b32e)